common/models/src/auth/auth_cache.rs

```diff
@@ -4,6 +4,31 @@ use std::time::{Duration, Instant};
 
 use parking_lot::Mutex;
 
+use super::user::UserInfo;
+
+#[derive(Clone, Hash, Eq, PartialEq)]
+pub struct AuthCacheKey {
+    pub authorization: String,
+    pub tenant: String,
+}
+
+impl AuthCacheKey {
+    pub fn new(user_info: &UserInfo, tenant: impl Into<String>) -> Self {
+        Self {
+            authorization: user_info.to_authorization(),
```

Comment:
There is a  BASE64 hashing here, in `UserInfo::to_authorization(&self)`, for each API callings.

1. Client send HTTP request with header `"Authorization": "Basic $(base64($username:$password))"`.
2. Server decode the HTTP header value: `username` and `password`.
3. Server encode the `username` and `password` into `"Basic $(base64($username:$password))"`.

I think the step 3 is needless.

Reply:
Let's solve this later。